google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/GeneratedUpdateTableStatementsTests.java

```diff
@@ -90,8 +90,10 @@ public void testUpdateStatements_createTables() throws SQLException {
   @Test
   public void testUpdateStatements_alterTables() throws SQLException {
     setupTestTables("Employee");
-
     List<String> sqlStrings = mockConnection.getStatementResultSetHandler().getExecutedStatements();
+    // The "alter table Employee ADD COLUMN" statements are no longer included in the generated
+    // schema when updating an existing table. This schema change was introduced from Hibernate
+    // 5.5.2 onwards.
```

Comment:
@mpeddada1 Thanks. It might be helpful to give some feedback to Hibernate library maintainers if this is undesirable/unexpected behavior of the library from your perspective.